[12.0][IMP] l10n_br_nfse: Implementando amount_financial e ajustando impressão dos valores de impostos retidos

marcelsavegnago opened the pull request:
Depends #1051 

cc @mileo @gabrielcardoso21 @luismalta @renatonlima

renatonlima committed:
l10n_br_account: alterada a visão dos dados adicionais da nota fiscal de entrada e saida, resolvido métodos onchange da localização, l10n_br_delivery: criado menu para veiculos, otimizada a visão de veiculos, l10n_brs: Adicionada a pasta test dos modulos da localização para implementação de testes

renatonlima committed:
l10n_br_sale e l10n_br_delivery: Resolvido compatibilidade entre os modulos, l10n_brs: adicionado a pasta test para a elaboração de testes

renatonlima committed:
alterado comentários dos arquivos *.py e mudado estilo de codificação para ficar mais limpo o código

renatonlima committed:
Removido '\' do código

renatonlima committed:
l10n_br_delivery corrigido importação

renatonlima committed:
l10n_br_stock: Adicionado nas condições de entregas o campo de despesa por conta de: 0 - Emitente, 1 - Emitente e Destinatário, 2 - Emitente/Destinatário e 9 - Sem Frete, l10n_br_account: Alterado a exportação da NFe para incluir o novo campo de frete por conta.

rvalyi committed:
[REF] more empty lines removal

rvalyi committed:
[IMP] l10n_br_delivery: do not blow out in case of missing fiscal parameters on picking when creating invoice

renatonlima committed:
l10n_br_delivery: limpeza e formatação do código

renatonlima committed:
Transferido os campos obrigatórios no documento fiscal implementado no l10n_br_delivery para o módulo l10n_br_account

renatonlima committed:
Transferido os campos obrigatórios no documento fiscal implementado no l10n_br_delivery para o módulo l10n_br_account

renatonlima committed:
[FIX] l10n_br_delivery, l10n_br_account: adicionado filtro nas regras de posições fiscais, [FIX] l10n_br_purchase

renatonlima committed:
[FIX] Alterado métodos de criação de nota fiscal no módulo l10n_br_delivery

renatonlima committed:
[FIX] removed fiscal_operation_id field

renatonlima committed:
formatação do código PEP8

renatonlima committed:
[MIGR] migração do modulo l10n_br_delivery

renatonlima committed:
[MIGR] trocadas chaves update_xml, init_xml por data e demo_xml por demo nos arquivos __openerp__.py de todos modulos da localização

renatonlima committed:
[MERGE] migração do módulo l10n_br_data_delivery para versão 7.0

renatonlima committed:
[FIX] rotina de validação da NFe do módulo l10n_br_delivery

renatonlima committed:
Alterado o nome de alguns módulos para deixar mais legível

renatonlima committed:
Adicionado arquivos de tradução vazios pt_BR.po para evitar mensagens de alertas no log e futura internacionalização dos termos da localização

renatonlima committed:
Atualização das versões dos módulos

renatonlima committed:
[REF] Refatorado aquivos no módulo l10n_br_delivery

renatonlima committed:
[REF] Refatorado aquivos de visões no módulo l10n_br_delivery

mileo committed:
[MIG] Migração do desconto e frete na branch develop, arquivos do l10n_br_delivery

mileo committed:
Revert "[MIG] Frete e desconto l10n_br_account"

This reverts commit 664cc98e8b65adfb11f5975bad6a67d1f47d806d.

mileo committed:
[FIX][REF] PEP8 class name, linha removida incorretamente

mileo committed:
[FIX] Class name

renatonlima committed:
[FIX] Alguns refactoring para concertar a geração da fatura quando não há um pedido de venda relacionado no módulo l10n_br_delivery e refatorado alguns métodos no l10n_br_account_product

renatonlima committed:
[FIX] corrigido nome da classe errado

danimaribeiro committed:
Corrigido erros de digitação e adicionado traduções aos módulos

renatonlima committed:
Inicio da extração dos campos e métodos do módulo l10n_br_sale para o novo módulo l10n_br_sale_product

renatonlima committed:
Inicio da extração dos campos e métodos do módulo l10n_br_delivery para o novo módulo l10n_br_sale_product

renatonlima committed:
Extraido métodos do módulo l10n_br_delivery para o novo módulo l10n_br_sale_product

renatonlima committed:
Movido objetos herdados da empresa do módulo l10n_br_delivery para o novo módulo l10n_br_sale_product

renatonlima committed:
Concluído a criação do módulo l10n_br_sale_product

renatonlima committed:
Adicionados arquivos README.md para descrição dos módulos

renatonlima committed:
[MIGRATION] initial commit to new branch 8.0 before migration

renatonlima committed:
update metadata info

Conflicts:
	l10n_br_delivery/__openerp__.py

renatonlima committed:
migrate stock.picking object and view to new API

renatonlima committed:
migrate delivery.carrier to new API

renatonlima committed:
migrate all objects in l10n_br_delivery.py to new API

renatonlima committed:
migrate all objects in sale.py to new API

renatonlima committed:
migrate all objects in stock.py to new API

renatonlima committed:
migrate stock objects

renatonlima committed:
[FIX] import

renatonlima committed:
Added api

renatonlima committed:
fix self.user to self.env.user

renatonlima committed:
fix stock.move and stock.picking objects

renatonlima committed:
removed debug

renatonlima committed:
migrate method to new api

renatonlima committed:
moved models and views to a folder as new pattern and replace uncoding to coding in py files

danimaribeiro committed:
Modificado interface dos transportes, criado onchange para setar os campos automaticamente quando existe um metodo cadastrado e um veículo

Conflicts:
	l10n_br_delivery/views/account_invoice_view.xml

sbidoul committed:
[MOV] move addons out of __unported__ (they remain not installable)

rvalyi committed:
pep8 hopefully cleared

mileo committed:
[FIX] CamelCase Classes

mileo committed:
[REF] Removing active tag from __openerp__.py

mileo committed:
[FIX] PEP8

[REF] -*- coding: utf-8 -*-

Squash

pep8

mileo committed:
[MIG] Migration l10n_br_delivery.py and stock.py

mileo committed:
[MIG] Migrate to new api delivery_set

mileo committed:
[FIX] Args delivery _get_invoice_line_vals

mileo committed:
[FIX] delivery UserError

renatonlima committed:
pep8

renatonlima committed:
chaged README file

renatonlima committed:
update author and version

renatonlima committed:
[REF] methods in stock.picking

mileo committed:
[REF] Replace tabs with spaces

find . -name '*.xml' ! -type d -exec bash -c 'expand -t 4 "$0" > /tmp/e && mv /tmp/e "$0"' {} \;

rvalyi committed:
lighter new style OCA license headers

renatonlima committed:
[REF] change view in l10n_br_delivery

renatonlima committed:
[MIG] Make modules uninstallable

renatonlima committed:
[MIG] Rename manifest files

mileo committed:
[NEW] Payment and Payment Lines

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[COPY] account.payment.term

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Split files

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

luismalta committed:
[ADD] fiscal_payment_ids onchange

luismalta committed:
[ADD] payment_term_id onchange

luismalta committed:
[FIX] financial_ids readonly

luismalta committed:
[ADD] compute change

mileo committed:
[FIX] Tests fiscal_payment

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Required fields in payment

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Document Payment term

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
WIP

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[ADD] Filling of fiscal_payment_ids and financial_ids

mileo committed:
[FIX] Clean code payment in fiscal

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

hendrixcosta committed:
[FIX] Flake8

hendrixcosta committed:
[FIX] remove print

mileo committed:
[IMP] Demo, tests and usability

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] Test coverage of financial

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Confirmação do documento com financeiro

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[FIX] Unlink of payment_term

mileo committed:
[REF] REM document_abstract.py

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Melhorias pos-rebase

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] _onchange_fiscal_operation_id

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[MOV] check/generate/onchange financial to l10n_br_fiscal.document.mixin.methods

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[IMP] DEMO DATA of fiscal payments

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Payment mixin

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Payment Condition / Payment Mode / Payment Term

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Payment mode attrs

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Flake8, Lint and tests

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[NEW] Adapting l10n_br_fiscal payment mode

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Payment Condition

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REM] Payment Mode

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Data data after rebase

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[REF] Remove payment from fiscal mixin

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Flake8

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] nfse financial tests

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

mileo committed:
[FIX] Runbot field with same label

Signed-off-by: Luis Felipe Mileo <mileo@kmee.com.br>

gabrielcardoso21 committed:
[REF] Rename module and fix dependencies

gabrielcardoso21 committed:
[ADD] Freight apportionment

DiegoParadeda committed:
[REF] Remove no longer used files and set v8 -> v12

DiegoParadeda committed:
[ADD] Disable add delivery line from sale.order and stock.picking

gabrielcardoso21 committed:
[FIX] Runtime error

DiegoParadeda committed:
[FIX] Version and headers

DiegoParadeda committed:
[FIX] Flake 8

DiegoParadeda committed:
[RMV] Files from l10n_br_delivery

DiegoParadeda committed:
[FIX] Flake8

gabrielcardoso21 committed:
[REM] Commented code

gabrielcardoso21 committed:
[ADD] Inverse function for other costs

gabrielcardoso21 committed:
[ADD] Inverse function for insurance value

luismalta committed:
[FIX] l10n_delivery manifest

luismalta committed:
[ADD] README files

DiegoParadeda committed:
[ADD] Readme and manifest

gabrielcardoso21 committed:
[IMP] Update values on save

luismalta committed:
[ADD] l10n_br_delivey tests

luismalta committed:
[FIX] amount_insurance and amount_costs readonly

DiegoParadeda committed:
[ADD] Tests

luismalta committed:
[FIX] Picking state on test

luismalta committed:
[IMP]  New test cases

luismalta committed:
[IMP] l10n_br_delivery test cases and performance

rvalyi committed:
overrideable create_m2o in _prepare_import_dict

useful for importing NFe emit as dest and avoiding to create
a broken related enderEmit for enderDest.

rvalyi committed:
license headers -> LGPL 3

britoederr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 1.6% (90 of 5394 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_base
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_base/pt_BR/

OCA-git-bot committed:
Merge PR #967 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update l10n_br_delivery.pot

oca-travis committed:
[UPD] Update l10n_br_hr.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

mileo committed:
[FIX] Module dependencies, typo and contributors

OCA-git-bot committed:
Merge PR #965 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update l10n_br_website_sale_delivery.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

luismalta committed:
[ADD] Create separeted invoices for each document operation line in the contract

luismalta committed:
[REF] l10n_br_contract tests

luismalta committed:
[FIX] Flake8

luismalta committed:
[REF] contract_fiscal_operation_id field name

luismalta committed:
[REF] Contract Line fiscal onchanges

luismalta committed:
[ADD] Fiscal operation and Fiscal Operation Line in Contract Line view

luismalta committed:
[REF] Contract Fiscal fields computation

mileo committed:
Added translation using Weblate (Portuguese (Brazil))

OCA-git-bot committed:
Merge PR #789 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update spec_driven_model.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

luismalta committed:
[IMP] Contract and contract line fiscal required fields

luismalta committed:
[REF] Fiscal Operation fields default behaviour

luismalta committed:
[ADD] l10n_br_contract demo data

OCA-git-bot committed:
Merge PR #911 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update l10n_br_contract.pot

oca-travis committed:
[UPD] Update l10n_br_hr.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5394 of 5394 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_base
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_base/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (35 of 35 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_zip
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_zip/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (11 of 11 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_contract
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_contract/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (18 of 18 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_crm
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_crm/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (31 of 31 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-payment_cielo
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-payment_cielo/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (42 of 42 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_resource
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_resource/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (107 of 107 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_nfse
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_nfse/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_currency_rate_update
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_currency_rate_update/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (22 of 22 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_website_sale
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_website_sale/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (20 of 20 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_portal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_portal/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (12 of 12 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_stock
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_stock/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (8 of 8 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_nfse_ginfes
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_nfse_ginfes/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (11 of 11 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_delivery
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_delivery/pt_BR/

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_website_sale_delivery
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_website_sale_delivery/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-spec_driven_model
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-spec_driven_model/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 4.6% (3 of 65 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_coa
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_coa/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

rvalyi committed:
make XML export schema agnostic

OCA-git-bot committed:
Merge PR #1046 into 12.0

Signed-off-by mileo

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 20.0% (13 of 65 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_coa
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_coa/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (42 of 42 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_resource
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_resource/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-spec_driven_model
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-spec_driven_model/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (35 of 35 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_zip
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_zip/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (11 of 11 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_delivery
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_delivery/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (12 of 12 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_stock
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_stock/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 99.6% (324 of 325 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (325 of 325 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_hr
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_hr/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1377 of 1377 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/pt_BR/

britoederr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (52 of 52 strings)

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_sale
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_sale/pt_BR/

marcelsavegnago committed:
[IMP] Ajustando DNFSe

marcelsavegnago committed:
[FIX] tamanho do campo discriminacao

gabrielcardoso21 committed:
[FIX] Core method call without product

marcelsavegnago committed:
[IMP] Implementando amount_financial

marcelsavegnago committed:
[IMP] Implementando amount_financial

marcelsavegnago committed:
[IMP] l10n_br_fiscal: Inclusão de ISSQN 2,5 e ISSQN RET 2,5

marcelsavegnago committed:
[FIX] correção do valor dos impostos retidos

marcelsavegnago committed:
[FIX] correção valor Retenções Federais

OCA-git-bot committed:
Merge PR #1048 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update l10n_br_nfse.pot

OCA-git-bot committed:
l10n_br_nfse 12.0.1.5.1

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_nfse
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_nfse/

mileo commented on l10n_br_nfse/models/document_line.py:
```diff
@@ -103,18 +103,18 @@ def prepare_line_servico(self):
-            'valor_inss': float(self.inss_value),
-            'valor_ir': float(self.irpj_value),
-            'valor_csll': float(self.csll_value),
+            'valor_pis': float(self.pis_wh_value),
+            'valor_cofins': float(self.cofins_wh_value),
+            'valor_inss': float(self.inss_wh_value),
+            'valor_ir': float(self.irpj_wh_value),
+            'valor_csll': float(self.csll_wh_value),
```
@marcelsavegnago não seria melhor criar entradas no dict como:

![image](https://user-images.githubusercontent.com/1975978/103088312-354e5080-45c9-11eb-849a-ba4ae7be9ea5.png)

marcelsavegnago replied:
No caso seria por exemplo ter o valor_pis e o valor_pis_retido?

marcelsavegnago replied:
@mileo olhando o fork da trust e outros leiautes de NFSe me pareceu que sempre que for informado o valor do pis, valor do cofins e afins os mesmos serão tratados como retidos. Ou estou enganado ? 

cc @jpcarassato

marcelsavegnago replied:
@mileo eu não entendi muito tua sugestão. Pode me ajudar ?

marcelsavegnago replied:
@mileo dá uma olhada.. acho que agora ficou melhor.
cc @gabrielcardoso21 @mbcosta @luismalta

OCA-git-bot committed:
Merge PR #1049 into 12.0

Signed-off-by mileo

OCA-git-bot committed:
Merge PR #1053 into 12.0

Signed-off-by mileo

OCA-git-bot committed:
l10n_br_fiscal 12.0.5.3.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #1050 into 12.0

Signed-off-by mileo

OCA-git-bot committed:
Merge PR #1051 into 12.0

Signed-off-by mileo

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 12.0.5.3.2

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/

oca-travis committed:
[UPD] Update l10n_br_fiscal.pot

OCA-git-bot committed:
l10n_br_fiscal 12.0.5.4.0

OCA-git-bot committed:
[UPD] addons table in README.md

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-brazil-12.0/l10n-brazil-12.0-l10n_br_fiscal
Translate-URL: https://translation.odoo-community.org/projects/l10n-brazil-12-0/l10n-brazil-12-0-l10n_br_fiscal/

marcos-mendez reviewed (commented):
No final das contas temos que configurar manualmente, certo @marcelsavegnago ?

marcelsavegnago commented:
> No final das contas temos que configurar manualmente, certo @marcelsavegnago ?

Neste momento estou testando diretamente a inclusão do documento fiscal.. mas vou checar como é identificado se deve ou não ser retido.

@mileo @renatonlima sabem informar ?

marcelsavegnago committed:
[FIX] melhorando a impressão dos tributos federais

DiegoParadeda committed:
[FIX] payment_cielo tests

OCA-git-bot committed:
Merge PR #1064 into 12.0

Signed-off-by rvalyi

marcos-mendez reviewed (approved):
Acho que ficou ok, obrigado Marcel
